packages/jest-console/src/matchers.js

```diff
@@ -69,8 +69,8 @@ expect.extend(
 
 		return {
 			...result,
-			[ matcherName ]: createToBeCalledMatcher( `.${ matcherName }`, methodName ),
-			[ matcherNameWith ]: createToBeCalledWithMatcher( `.${ matcherNameWith }`, methodName ),
+			[ matcherName ]: createToHaveBeenCalledMatcher( `.${ matcherName }`, methodName ),
+			[ matcherNameWith ]: createToHaveBeenCalledWith( `.${ matcherNameWith }`, methodName ),
 		};
 	}, {} )
 );
```

Comment:
Should the changes here for `jest-console` be added to the changelog?

Reply:
It was part of rename operation and I figured out at the end that it's an internal change. I kept it only for consistency :)